packages/gap-web-ui/package.json

```diff
@@ -29,26 +29,16 @@
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^13.3.0",
     "@rollup/plugin-typescript": "^8.3.3",
-    "@storybook/addon-actions": "7.4.6",
-    "@storybook/addon-essentials": "7.4.6",
-    "@storybook/addon-interactions": "7.4.6",
-    "@storybook/addon-links": "7.4.6",
     "@storybook/cli": "7.4.6",
-    "@storybook/preset-scss": "1.0.3",
```

Comment:
Can you leave these in as well? If you look at [this file](https://github.com/cabinetoffice/gap-find-apply-web/blob/develop/packages/gap-web-ui/.storybook/main.js) you can see they're all added to the storybook config (and then presumably imported by storybook when it runs).

Reply:
And they're back!